### PR TITLE
Adding missing usages and fields in function!

### PR DESCRIPTION
**Pull request by devmike123**

### CODE REVIEW

**Comment by devmike123** [comment]
@Leref can you review the pr

File: src/guide/extras/customFunctions.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
 # Custom Functions
-## Creating the custom function
-```javascript
+
+Custom functions are used to create special functions with custom code that can only be used by the specific client.
+
+### Usage
+```js
```
Comment (Leref) [inline src/guide/extras/customFunctions.md]: Put this as JavaScript.

File: src/guide/extras/customFunctions.md [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+
+{% hint style="info" %} To get the values of the parameters provided in the function, use {} around the parameter name i.e. {param}  {% endhint %}
+
+## Examples
+
+### Example with aoi.js function
+
+- Creating the custom function
+
+```js
```
Comment (Leref) [inline src/guide/extras/customFunctions.md]: it should be JavaScript syntax

File: src/guide/extras/customFunctions.md [hunk 3]
```diff
@@ -11,25 +39,24 @@ code : `
...
 ```
 
-{% hint style="info" %}
-WITH THIS FUNCTION MANAGER IT JUST CREATED `$say` FUNCTION
-{% endhint %}
+- Using the custom function
 
-## Using the custom function
-```javascript
+```js
```
Comment (Leref) [inline src/guide/extras/customFunctions.md]: Same thing here for JavaScript syntax.

**Review by Leref — CHANGES_REQUESTED** [review]
Since gitbook changed a while ago, it won't understand shortcut of `js` it should be `javascript`.

**Comment by devmike123** [comment]
@Leref fixed!

**Review by Leref — APPROVED** [review]

File: src/functions/usdban.md [hunk 4]
```diff
@@ -9,15 +9,15 @@ description: Bans an user from the guild using their ID.
 ### Usage
 
 ```php
-$ban[user id;guild id?;messages to delete?;reason?]
+$ban[guild id;user id;messages to delete?;reason?]
```
Comment (Leref) [inline src/functions/usdban.md]: No space when

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/functions/usdisbanned.md [hunk 5]
```diff
@@ -4,16 +4,24 @@ description: Checks if user is banned. Returns true or false
...
 # $isBanned
 
-This  function checks if the given user is banned or not. Returns boolean
+This function checks if the given user is banned or not. Returns boolean
 
-```text
-$isBanned[user ID;guild ID (optional)]
+### Usage
+```php
+$isBanned[guild ID;user ID]
```
Comment (Leref) [inline src/functions/usdisbanned.md]: No space when

File: src/functions/usdisservermuted.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+description: Checks if server is muted by the specified user. Returns true or false
+---
+
+# $isServerMuted
+
+This function checks if the given user has muted the server. Returns boolean
+
+### Usage
+```php
+$isServerMuted[guild ID;user ID]
```
Comment (Leref) [inline src/functions/usdisservermuted.md]: Revert. GuildID last field.

File: src/guide/extras/customFunctions.md [hunk 7]
```diff
@@ -1,5 +1,33 @@
 # Custom Functions
-## Creating the custom function
```
Comment (Leref) [inline src/guide/extras/customFunctions.md:2]: Revert.

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/functions/usdisservermuted.md (hunk 6 above)
Comment (devmike123) [reply src/functions/usdisservermuted.md]: Done.

File: src/guide/extras/customFunctions.md (hunk 7 above)
Comment (devmike123) [reply src/guide/extras/customFunctions.md:2]: Why?

Comment (Leref) [reply src/guide/extras/customFunctions.md:2]: It's just the title of a page, it shouldn't be too long.
Straightforward.

Comment (devmike123) [reply src/guide/extras/customFunctions.md:2]: Ok changing it.

**Review by Leref — APPROVED** [review]

**Pull request merged**